cluster-install/install.sh

```diff
@@ -493,6 +498,21 @@ get_limits_list_fancy() {
 #----------------------------------------------------------------------
 #-- Util functions
 #----------------------------------------------------------------------
+get_captured_err() {
+    if [ -f "$__ERR_FILE" ]; then
+        cat "$__ERR_FILE"
+        rm -f "$__ERR_FILE" &> /dev/null
```

Comment:
Trying to make this ERR_FILE handling as behind-the-scenes as possible, so it's cleaned up every time we read its value. In the event the script exits before its value is read, there is a cleanup function later on that gets called anyway.

One exception is if we run into a `fatal` error (which ideally will never happen to a customer as those should be reserved for dev-specific guardrails), in which case we deliberately leave the file alone for debugging purposes.